Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: test test-integration build run proto-api proto-lint clean download install-tools generate check-go-version
 
-VERSION=`./scripts/get-tag.sh`
+VERSION=`git describe --tags --dirty`
```

Comment:
The reason why I used the script with `git diff-index` was that not all changes I made locally were being detected by `git describe`. I'll try to find what those changes were.

Reply:
IIRC the difference was to detect untracked files, but that seems to be a bit too strict for Docker.

Reply:
Right... Personally, I'd like the tag to still be dirty, when files are added (simply to be sure there are no unintentional commits or such). On the other hand, everything which already got into GitHub is tracked, so whatever is untracked can only be generated files, which we don't really care about.

Reply:
Btw, if you don't mind leaving a comment explaining why we have the command as it is, that'd be really useful.

Reply:
I added a doc above the command 👍